### ISSUE TITLE
newlines_after_classes false positive with simple class

### ISSUE THREAD

**Issue by dorianj** [clutchski/coffeelint#347]
A file with just this:
```coffeescript
class WhitespaceTest
  key1: 'a'
  key2: 'b'

```
triggers `newlines_after_classes` for me.

(my relevant config):
```json
    "newlines_after_classes": {
        "name": "newlines_after_classes",
        "value": 2,
        "level": "error"
    }
```

Things that work-around this: 
 * adding a function prop anywhere in the object (`b: ->`)
 * adding an extra newline, or comment, at end of page
 * only having a single property

**Comment by KODerFunk** [comment]
Similar reports #245
